include/faabric/transport/PointToPointServer.h

```diff
@@ -11,7 +11,7 @@ class PointToPointServer final : public MessageEndpointServer
     PointToPointServer();
 
   private:
-    PointToPointBroker& reg;
+    PointToPointBroker& broker;
```

Comment:
This used to be called a "registry" hence the `reg` name, but it was a bit confusing.